src/components/TabSelector/TabSelectorItem.tsx

```diff
@@ -50,39 +57,61 @@ function TabSelectorItem({
     testID,
+    shouldShowProductTrainingTooltip = false,
+    renderProductTrainingTooltip,
 }: TabSelectorItemProps) {
     const styles = useThemeStyles();
     const [isHovered, setIsHovered] = useState(false);
 
-    return (
-        <Tooltip
-            shouldRender={!shouldShowLabelWhenInactive && !isActive}
-            text={title}
+    const shouldShowEducationTooltip = shouldShowProductTrainingTooltip && isActive;
+
+    const children = (
+        <AnimatedPressableWithFeedback
+            accessibilityLabel={title}
+            style={[styles.tabSelectorButton, styles.tabBackground(isHovered, isActive, backgroundColor), styles.userSelectNone]}
+            wrapperStyle={[styles.flexGrow1]}
+            onPress={onPress}
+            onHoverIn={() => setIsHovered(true)}
+            onHoverOut={() => setIsHovered(false)}
+            role={CONST.ROLE.BUTTON}
+            dataSet={{[CONST.SELECTION_SCRAPER_HIDDEN_ELEMENT]: true}}
+            testID={testID}
         >
-            <AnimatedPressableWithFeedback
-                accessibilityLabel={title}
-                style={[styles.tabSelectorButton, styles.tabBackground(isHovered, isActive, backgroundColor), styles.userSelectNone]}
-                wrapperStyle={[styles.flexGrow1]}
-                onPress={onPress}
-                onHoverIn={() => setIsHovered(true)}
-                onHoverOut={() => setIsHovered(false)}
-                role={CONST.ROLE.BUTTON}
-                dataSet={{[CONST.SELECTION_SCRAPER_HIDDEN_ELEMENT]: true}}
-                testID={testID}
-            >
-                <TabIcon
-                    icon={icon}
+            <TabIcon
+                icon={icon}
+                activeOpacity={styles.tabOpacity(isHovered, isActive, activeOpacity, inactiveOpacity).opacity}
+                inactiveOpacity={styles.tabOpacity(isHovered, isActive, inactiveOpacity, activeOpacity).opacity}
+            />
+            {(shouldShowLabelWhenInactive || isActive) && (
+                <TabLabel
+                    title={title}
                     activeOpacity={styles.tabOpacity(isHovered, isActive, activeOpacity, inactiveOpacity).opacity}
                     inactiveOpacity={styles.tabOpacity(isHovered, isActive, inactiveOpacity, activeOpacity).opacity}
                 />
-                {(shouldShowLabelWhenInactive || isActive) && (
-                    <TabLabel
-                        title={title}
-                        activeOpacity={styles.tabOpacity(isHovered, isActive, activeOpacity, inactiveOpacity).opacity}
-                        inactiveOpacity={styles.tabOpacity(isHovered, isActive, inactiveOpacity, activeOpacity).opacity}
-                    />
-                )}
-            </AnimatedPressableWithFeedback>
+            )}
+        </AnimatedPressableWithFeedback>
+    );
+
+    return shouldShowEducationTooltip ? (
+        <EducationalTooltip
+            shouldRender={shouldShowEducationTooltip}
```

Comment:
```suggestion
            shouldRender
```
this will always be true, since we only pick this if `shouldShowEducationTooltip`